receiver/databricksreceiver/metadata.yaml

```diff
@@ -0,0 +1,1223 @@
+name: databricksreceiver
+
+status:
+  type: receiver
+  stability: development
+  pipelines: [metrics]
+  distributions: [contrib]
+
+resource_attributes:
+  databricks.instance.name:
+    description: The name of the Databricks instance as defined by the value of the "instance_name" field in the config
+    type: string
+  spark.cluster.id:
+    description: The ID of the Spark cluster
+    type: string
+  spark.cluster.name:
+    description: The name of the Spark cluster
+    type: string
+  spark.app.id:
+    description: The ID of the Spark cluster app
+    type: string
+attributes:
+  job.id:
+    description: The numeric ID of the Databricks job
+    type: int
+  task.id:
+    description: The name of the Databricks task
+    type: string
+  cluster.id:
+    description: The ID of the Databricks cluster
+    type: string
+  spark.app.id:
+    description: The ID of the Spark application
+    type: string
+  spark.executor.id:
+    description: The ID of the Spark executor
+    type: string
+  pipeline.id:
+    description: The ID of the Databricks pipeline
+    type: string
+  pipeline.name:
+    description: The name of the Databricks pipeline
+    type: string
+  spark.job.id:
+    description: The ID of the Spark job
+    type: int
+  task.type:
+    description: The type of the Databricks task
+    type: string
+    enum:
+      - NotebookTask
+      - SparkJarTask
+      - SparkPythonTask
+      - PipelineTask
+      - PythonWheelTask
+      - SparkSubmitTask
+metrics:
+  databricks.jobs.total:
+    enabled: true
+    description: A snapshot of the total number of jobs registered in the Databricks instance taken at each scrape
+    unit: "{jobs}"
+    gauge:
+      value_type: int
+  databricks.jobs.schedule.status:
```

Comment:
Is this metric just an aggregation of `databricks.tasks.schedule.status`? If that's the case, I have a couple of Qs:

1. Do we need to expose them both? If so, we should make one of them optional, otherwise, we expose duplicated data (just in different forms). Any backend (or collector) can do the aggregation.
2. If the aggregation method is sum, they should be sums, not gauges.

Reply:
The status comes from the job, and a job has multiple tasks, so the job status is simply propagated to each of its tasks. I suppose the task status metric could be made optional or removed altogether. What do you think?